### PR TITLE
record access mode mobile support

### PR DESCRIPTION
**Pull request by elsiosanchez**
## Feature
record access mode mobile support
#### Screenshot or Gif
Peek 16-04-2021 00-27

#### Link to minimal reproduction
[demo-ui-erpya](https://deploy-preview-755--stoic-lamport-68decf.netlify.app/)

#### Other relevant information
- Your OS:Debian9
- Web Browser:Chrome
- Node.js version:10.9

### CODE REVIEW

**Comment by yamelsenih** [comment]
Two comments:
- For desktop the access dialog is not showed
- The **Action** and **References** button is loaded after click evend but the **Relation** button is showed after over event
Peek 2021-04-17 07-25

**Comment by oscarinak** [comment]
A review of the case was made where it is observed that part of the "Access to Record" option in the ADempiere window its fields adapts to the mobile view, however, as users are added either to "Allow Viewing" or " Hide Registry "the users that are added in the window to check if it is read only or dependent entities these are not displayed as they are loaded or added.

Error_Acceso_a_Registros

It is also noted that there is no way to delete the associated users

File: src/components/ADempiere/RecordAccess/recordAccessMobile.vue [hunk 1]
```diff
@@ -0,0 +1,281 @@
+<template>
+  <el-card class="box-card">
+    <div slot="header" class="clearfix">
+      <span>
+        Acceso A registros
```
Comment (yamelsenih) [inline src/components/ADempiere/RecordAccess/recordAccessMobile.vue]: Please add translatioon for this

File: src/components/ADempiere/RecordAccess/recordAccessMobile.vue [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+      </span>
+    </div>
+    <div style="margin-bottom: 10%;">
+      <span style="margin-bottom: 10%;">
+        Ocultar Registro
```
Comment (yamelsenih) [inline src/components/ADempiere/RecordAccess/recordAccessMobile.vue]: Please add translatioon for this

File: src/components/ADempiere/RecordAccess/recordAccessMobile.vue [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+        />
+      </el-select>
+    </div>
+    <div
+      style="margin-bottom: 10%;"
+    >
+      <span
+        style="margin-bottom: 10%;"
+      >
+        Permitir Visualizacion del Registro
```
Comment (yamelsenih) [inline src/components/ADempiere/RecordAccess/recordAccessMobile.vue]: Please add translatioon for this

**Review by yamelsenih — CHANGES_REQUESTED** [review]
Please add translatioon for this

**Pull request merged**